pipenv/core.py

```diff
@@ -785,14 +785,14 @@ def do_install_dependencies(
     """"
     Executes the install functionality.
 
-    If requirements is True, simply spits out a requirements format to stdout.
+    If emit_requirements is True, simply spits out a requirements format to stdout.
     """
 
     from six.moves import queue
-    if requirements:
+    if emit_requirements:
         bare = True
-    # Load the lockfile if it exists, or if only is being used (e.g. lock is being used).
-    if skip_lock or only or not project.lockfile_exists:
+    # Load the lockfile if it exists, or if dev_only is being used.
+    if skip_lock or dev_only or not project.lockfile_exists:
```

Comment:
@techalchemy @frostming I'm not sure why `dev_only` (formerly called `only`) was ever in this conditional. To get PEEP 6 to work as intended, I think I may actually need to remove it from, and follow the same branch as the regular non-dev requirements.